app/javascript/dashboard/components/ui/WootButton.vue

```diff
@@ -14,7 +14,9 @@
       :icon="icon"
       :icon-size="iconSize"
     />
-    <span v-if="$slots.default" class="button__content"><slot /></span>
+    <span v-if="$slots.default" class="button__content text-truncate">
```

Comment:
Instead of having this by default, Is it possible to use the `text-truncate` with `<w-btn class="text-truncate">` ?

Reply:
Sure will update

Reply:
@nithindavid It's not working when we add `text-truncate` class in the `<w-btn class="text-truncate">`. Because the button content is inside the `span` tag.

<img width="933" alt="image" src="https://user-images.githubusercontent.com/64252451/187121760-4763468d-3254-4ba8-8c71-3958955c45ce.png">

Reply:
We could stick to "Yes, delete portal" and make the portal name in body bold.

Also align the buttons to right inside the modal and keep the delete button on right side from left side

Reply:
@nithindavid Update the PR 
<img width="699" alt="image" src="https://user-images.githubusercontent.com/64252451/187458906-aa358f73-a705-47d0-8ab2-13a5b20abb4f.png">